scripts/install-deeplearning-libraries.sh

```diff
@@ -64,16 +64,14 @@ fi
 # You might need to install liblapack-dev package as well
 # sudo apt-get install -y liblapack-dev
 if [ ! -d "numpy" ]; then
```

Comment:
This should probably be changed to:

```bash
 if [ ! -f "~/.numpy-site.cfg" ]; then
```
and then the `pip install numpy` command should be moved to after the `fi` to only protect the symlink creation and let the pip upgrade numpy to the latest version if possible with:

```bash
pip install --upgrade numpy
```